pkg/webui/console/views/overview/index.js

```diff
@@ -58,6 +58,7 @@ const m = defineMessages({
   createGateway: 'Register a gateway',
   gotoApplications: 'Go to applications',
   gotoGateways: 'Go to gateways',
+  needHelp: 'Need help? Have a look at our {documentationLink} or {supportLink}',
   welcome: 'Welcome to the Console!',
   welcomeBack: 'Welcome back, {userName}! 👋',
   getStarted: 'Get started right away by creating an application or registering a gateway',
```

Comment:
```suggestion
  getStarted: 'Get started right away by creating an application or registering a gateway.',
```
Please change L64 accordingly as well.